problems/5.loops.js

```diff
@@ -2,17 +2,26 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
-
+function evenNumbers(num) {
+  let arr=[]
+  for (let num=0; num < number.length ;num +=1)
```

Comment:
What is number ? Your parameter is called num, so don't want to redeclare num in side of for loop.